assets/methods_description_template.yml

```diff
@@ -3,3 +3,23 @@ description: "Suggested text and references to use when describing pipeline usag
 section_name: "Arcadia-Science/hifi2genome Methods Description"
 section_href: "https://github.com/Arcadia-Science/hifi2genome"
 plot_type: "html"
+data: |
+  <h4>Methods</h4>
+  <p> Data was processed using Arcadia-Science/hifi2genome v${workflow.manifest.version} ${doi_text}.
+  <p> The pipeline was executed with Nextflow v${workflow.nextflow.version} (<a href="https://doi.org/10.1038/nbt.3820">[3]</a>) with the following command:</p>
+  <pre><code>${workflow.commandLine}</code></pre>
+  <h4>References</h4>
+  <ul>
+    <li> [INSERT REF]
```

Comment:
?

Reply:
oh i get it, as a placeholder for the hifi2genome pub?

Reply:
yes!